pkg/providers/vault/config.go

```diff
@@ -22,8 +22,9 @@ import (
 
 // SecretConfig contains configuration settings used to communicate with an HTTP based secret provider
 type SecretConfig struct {
-	Host                    string
-	Port                    int
+	Host string
```

Comment:
This change might trip gofmt.  Better run `gofmt -l .` and check before submitting the real PR.

Reply:
gofmt -l is good.